basic_hashtable/b_hashtables.c

```diff
@@ -84,7 +88,15 @@ BasicHashTable *create_hash_table(int capacity)
  ****/
 void hash_table_insert(BasicHashTable *ht, char *key, char *value)
 {
+  int index = hash(key, (ht->capacity - 1));
+
+  if (ht->storage[index] != NULL){
+    printf("Warning: You have overwritten \"%s\n\"", ht->storage[index]->value);
+    free(ht->storage[index]);
```

Comment:
I think `destroy_pair(ht->storage[index]);` here

Reply:
Oh I definitely missed that one. I'll change it.